release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -692,6 +692,22 @@ A platform Operator is an OLM-based Operator that can be installed during or aft
 
 For more information on platform Operators, see xref:../operators/admin/olm-managing-po.adoc#olm-managing-po[Managing platform Operators]. For more information on RukPak and its resources, see xref:../operators/understanding/olm-packaging-format.adoc#olm-rukpak-about_olm-packaging-format[Operator Framework packaging format].
 
+[id="ocp-4-12-olm-operator-node-affinity"]
+==== Controlling where an Operator is installed
+
+By default, when you install an Operator, OpenShift Container Platform installs the Operator pod to one of your worker nodes randomly.
+
+In {product-title} 4.12, you can control where an Operator pod is installed by adding node affinity or pod anti-affinity constraints to the Operator’s Subscription object.
```

Comment:
I think here we either want to just say affinity constraints or we want to list out node affinity, pod affinity, or pod anti-affinity. Otherwise it's a strange omission.

Reply:
Fixed!